m3u8/model.py

```diff
@@ -817,7 +817,7 @@ def __init__(self, uri, stream_info, media, base_uri):
         self.base_uri = base_uri
 
         resolution = stream_info.get('resolution')
-        if resolution is None:
+        if resolution != None:
```

Comment:
it should be `resolution is not None`? also isn't this related to your other open PR, I think you must split the code changes.